x/stake/keeper/key.go

```diff
@@ -177,28 +177,28 @@ func GetREDByValDstIndexKey(delAddr sdk.AccAddress, valSrcAddr, valDstAddr sdk.V
 	return key
 }
 
-// rearranges the ValSrcIndexKey to get the REDKey
+// GetREDKeyFromValSrcIndexKey rearranges the ValSrcIndexKey to get the REDKey
 func GetREDKeyFromValSrcIndexKey(IndexKey []byte) []byte {
```

Comment:
I know this wasn't your code, but can we correctly name the parameters, `indexKey`?

Reply:
Sure, I'll address this! If this is something we want to enforce, I think we should add it to the linter. Perhaps open up a follow-up issue for fixing the general case?

Reply:
Yes please!